lib/utils/ember.js

```diff
@@ -298,16 +298,55 @@ function isObserverProp(node) {
   return isPropOfType(node, 'observer');
 }
 
-function isComputedProp(node) {
+function isComputedProp(
+  node,
+  importedEmberName,
+  importedComputedName,
+  { includeSuffix, includeMacro } = {}
+) {
   const allowedMemberExpNames = new Set(['volatile', 'readOnly', 'property', 'meta']);
```

Comment:
We could extract this set outside of the function, so that don't have to keep creating it.

Reply:
Fixed.